client/src/pages/ProfileSkeleton/ProfileEditionMock/ProfileEditionMock.tsx

```diff
@@ -0,0 +1,41 @@
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import ProfileSelection from '../../../components/ProfileSelection/ProfileSelection';
+
+export default function ProfileEditionMock(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Grid container className={classes.root}>
+      <Grid item xs={6} sm={6} md={5} elevation={0} component={Paper} square className={classes.back}>
+        <Box className={classes.authWrapper}>
+          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+            <ProfileSelection name={'Edit profile'} link={'/profileEditionMock'} selected={true} />
```

Comment:
```suggestion
            <ChatSideBanner name={'Edit profile'} link={'/profileEditionMock'} selected={true} />
```

Was this supposed to be `ChatSideBanner`? I don't see the `ProfileSelection` component being declared in this PR :)